Added pdf builders logging

MarosGrm opened the pull request.

leiblix commented:
Bylo by možné ty místa kde se testuje, že to není prázdné přesunout až přímo do generování PDF? 

Myslím, že by to bylo o něco čistější (když například někdo napíše builder pro jiný model). Nechci, aby se z těch builderů stal špagetový kód...

MarosGrm commented:
Dalo by sa to, ale takto vidíš v logoch priamo ten údaj ktorý tam chýba.

leiblix commented:
Jo, ale uz ted jsou dve mista kde se to muze nachazet. Budes do logu psat vzdy vsechny varianty?

leiblix commented:
Kromě toho je problematické, že v tom logu není vidět PID - nevíš který dokument je rozbitý.

MarosGrm committed:
Revert "Revert "Added pdf builders logging""

This reverts commit 84fc54b45ce059800444ac421f733eec6e9d73a9.

MarosGrm committed:
Revert "Added pdf builders logging"

This reverts commit ecab36f54ae29241c3c7d9d0a7192daf0dab990f.

MarosGrm committed:
Added pdf builders logging


added logging when pdf values are empty


Added logging for empty fields when generating PDF


Revert "Added pdf builders logging"

This reverts commit 11e0f12178f4b89caa1c7b0d8c19561562bea2f8.

ss

MarosGrm committed:
#198

leiblix commented:
Ty novější commity, to je rovnou přehození těch builderů? Funguje to? Co ty logy, ty tam nejsou :-)?

MarosGrm committed:
fix builders


fix builders

MarosGrm commented:
Ano, len AuthorBuilder je nutný ešte prerobiť. Čo sa logovania týka, tak už si to presne nepametám, ale viem, že tam bol problém, s tím, že ono to prechádza všetky buildre a loguje to potom aj veci ktoré s daným modelom nesúvisia. Revertol som commit, kde som to logovanie odstránil a idem to u seba ešte poriadne otestovať.
